applications/nrf_desktop/README.rst

```diff
@@ -334,12 +334,14 @@ Requirements
 
 The project comes with configuration files for the following boards:
 
-    * |nRF52840DK| - a sample where the mouse is emulated using DK buttons
-    * PCA20041 - nRF Desktop gaming mouse reference design
-    * PCA20037 - nRF Desktop keyboard reference design
-    * PCA20044 - nRF Desktop casual mouse reference design (nRF52832)
-    * PCA20045 - nRF Desktop casual mouse reference design (nRF52810)
-    * PCA10059 - nRF Desktop dongle reference design
+    * PCA10056 - the application is configured to work as a gaming mouse (motion emulated using DK buttons)
+    * PCA20041 - the application is configured to work as a gaming mouse
+    * PCA20037 - the application is configured to work as a keyboard
+    * PCA20044 - the application is configured to work as a casual desktop mouse (nRF52832 based board)
+    * PCA20045 - the application is configured to work as a casual desktop mouse (nRF52810 based board)
+    * PCA10059 - the application is configured to work as a HID dongle (nRF52840 based board)
+    * PCA10111 - the application is configured to work as a HID dongle (nRF52833 based board)
+    * PCA10100 - the application is configured to work as a HID dongle
```

Comment:
```suggestion
    * nRF52840 DK (PCA10056) - the application is configured to work as a gaming mouse (motion emulated using DK buttons)
    * nRF52840 Gaming Mouse (PCA20041) - the application is configured to work as a gaming mouse
    * nRF52832 Desktop Keyboard (PCA20037) - the application is configured to work as a keyboard
    * nRF52832 Desktop Mouse (PCA20044) - the application is configured to work as a casual desktop mouse 
    * nRF52810 Desktop Mouse (PCA20045) - the application is configured to work as a casual desktop mouse 
    * nRF52840 USB Dongle (PCA10059) - the application is configured to work as a HID dongle
    * nRF52833 USB Dongle (PCA10111) - the application is configured to work as a HID dongle
    * nRF52833 DK (PCA10100) - the application is configured to work as a HID dongle
```
I think we should use the official names. @jdstapes ?

Reply:
This is fine with me. I will squash and rebase